microsoft-edge/hosting/WebView2.md

```diff
@@ -18,7 +18,7 @@ The WebView2 control is currently in developer preview, during which you can pro
 
 ## Supported Platforms
 
-A developer preview is available for Win32 C++ on Windows 10, Windows 8.1, Windows 8, and Windows 7. An Alpha version for WinUI 3.0 is available [here](https://docs.microsoft.com/uwp/toolkits/winui3/). In the future, we plan to support WebView2 on .NET.  
+A developer preview is available for Win32 C++ on **Windows 10**, **Windows 8.1**, **Windows 8**, **Windows 7**, **Windows Server 2016**, **Windows Server 2012/2012R2**, and **Windows Server 2008 R2**. An Alpha version for WinUI 3.0 is available [here](https://docs.microsoft.com/uwp/toolkits/winui3/). In the future, we plan to support WebView2 on .NET.  
```

Comment:
Can you remove the bold from the product names please? Per MS Style guidelines, bold should be used for product UI text. Would using a bulleted list of the products allow for easier scanning?